db/migrate/20200224081312_create_deployments.rb

```diff
@@ -0,0 +1,13 @@
+class CreateDeployments < ActiveRecord::Migration[5.2]
+  def change
+    create_table :deployments do |t|
+      t.references :cluster, null: false
+      t.string  :name,       null: false
+      t.integer :count,      null: false, default: 1
```

Comment:
count is a reserved keyword in ruby, to make it more clear can we rename `desired_num_replica` for example?

Reply:
Noted. Yeah, it'll be more clear if we can rename that.

Reply:
We have changed this in another migration.